man/rustdoc.1

```diff
@@ -60,55 +59,50 @@ pass arguments to the test runner
 \fB\-\-html\-in\-header\fR \fIFILE\fR
 file to add to <head>
 .TP
-\fB\-\-html\-before\-content\fR \fIFILE\fR
-file to add in <body>, before content
+\fB\-\-html\-before\-content\fR \fIFILES\fR
+files to include inline between <body> and the content of a rendered Markdown
+file or generated documentation
 .TP
-\fB\-\-html\-after\-content\fR \fIFILE\fR
-file to add in <body>, after content
+\fB\-\-markdown\-before\-content\fR \fIFILES\fR
+files to include inline between <body> and the content of a rendered
+Markdown file or generated documentation
 .TP
-\fB\-\-markdown\-css\fR \fIFILE\fR
-CSS files to include via <link> in a rendered Markdown file
+\fB\-\-html\-after\-content\fR \fIFILES\fR
+files to include inline between the content and </body> of a rendered
+Markdown file or generated documentation
+.TP
+\fB\-\-markdown\-after\-content\fR \fIFILES\fR
+files to include inline between the content and </body> of a rendered
+Markdown file or generated documentation
+.TP
+\fB\-\-markdown\-css\fR \fIFILES\fR
+CSS files to include via <link> in a rendered Markdown file Markdown file or
+generated documentation
 .TP
 \fB\-\-markdown\-playground\-url\fR \fIURL\fR
 URL to send code snippets to
 .TP
 \fB\-\-markdown\-no\-toc\fR
 don't include table of contents
 .TP
-\fB\-h\fR, \fB\-\-help\fR
-Print help
+\fB\-\-markdown\-no\-toc\fR
+don't include table of contents
+.TP
+\fB\-h\fR, \fB\-\-extend\-css\fR
+to redefine some css rules with a given file to generate doc with your own theme
 .TP
 \fB\-V\fR, \fB\-\-version\fR
 Print rustdoc's version
 
 .SH "OUTPUT FORMATS"
 
-The rustdoc tool can generate output in either an HTML or JSON format.
+The rustdoc tool can generate output in either an HTML or Markdown format.
```

Comment:
Rustdoc can't create Markdown files, can't it? I was under the impression that the only thing rustdoc can output is the regular HTML pages.